lua/packer/clean.lua

```diff
@@ -25,7 +25,9 @@ local clean_plugins = function(_, plugins, results)
     results.removals = results.removals or {}
     await(a.main)
     local opt_plugins, start_plugins = plugin_utils.list_installed_plugins()
-    local missing_plugins = plugin_utils.find_missing_plugins(plugins, opt_plugins, start_plugins)
+    local missing_plugins = {}
+    local r = await(plugin_utils.find_missing_plugins(plugins))
+    missing_plugins = r or {}
```

Comment:
Nitpick, but maybe we shorten this to `local missing_plugins = await(plugin_utils.find_missing_plugins(plugins)) or {}`?